src/games/strategy/engine/data/ChangeFactory.java

```diff
@@ -105,6 +105,7 @@ public static Change changeResourcesChange(final PlayerID player, final Resource
   }
 
   /** TODO: unused code, kept per @veqryn */
+  @SuppressWarnings("unused")
```

Comment:
You're assuming the requirements of the function will be identical in a  future scenario? Could you add the unit test for this code as well so then someone really could just uncomment it?

My point is, remove unused code. If someone really needs it, they'll find it in the history. Otherwise if someone really needs this functionality, chances are really good they'll re-implement it and not discover that this function was here anyways. In this example, I don't mind too much since there is a TODO to at least indicate it is not actually used. 

If we really want to keep code samples around, a wiki is actually a better place. At least there we won't have to maintain dead code when we refacgtor stuff.